### DISCUSSION TITLE
希望在ios分享面板添加沉浸閲讀功能

### DISCUSSION THREAD

**Discussion by makisekurisu-jp** [immersive-translate/immersive-translate#1048] [Ideas]
### 新功能 | New Feature

![688d6687-9e57-4dbb-b79a-f106d642e02e](https://github.com/immersive-translate/immersive-translate/assets/72290820/7b4f09a8-6c1b-4f5a-9e0d-f7fc09cb88a1)

### 为什么您想要添加这些功能 | Why you want to add these features

使用netnewswire訂閲文章，打開分享面板只能open in browser后再使用沉浸翻譯，如果在分享面板添加功能按鍵就可以在app内閲讀，不需要額外開啓safari。

### 提议 | Proposal

希望作者可以實現這個功能需求，感謝！

**Comment by theowenyoung** [comment]
好建议～ 但是实现这个的成本目前来说有点高～

希望未来可以支持～

**Reply by makisekurisu-jp** [reply]
期待您的貢獻☺️，這將使得rss閱讀程式更方便翻譯閱讀。
